### DISCUSSION TITLE
wLuna from SOL to ETH Stuck

### DISCUSSION THREAD

**Discussion by thesighguy** [wormhole-foundation/wormhole#460] [Problems / bugs] [answered]
I used wormhole to successfully bridge Luna from ETH to SOL. When I tried to pull it back days later, it got stuck. The transaction seemed to be successful on SOL (I pulled up the tx), but nothing showed up on ETH side. I never got a prompt to verify the transaction in MM. And yes, I have enough ETH for gas. Please help.

Also, I checked for weird bridge tokens in my ETH wallet. No new tokens, so this isn't a w(wormhole)wLuna problem. It is definitely a "token didn't show up at all" problem.

I also tried waiting on v1 for the "recent transactions" data to show up so I could hit the retry button. There is no data, and the retry button never showed up. I tried the recovery tx input on v2. I got an error saying it could not recognize the tx number I put in.

https://solscan.io/tx/5zDv7KphnxMSeEuz8VdgHuwuNMboez9n8f1zBtjoZR849gNw19MgXcfaAqG1BYGUveLLJ3YrUBUMjsqNZXtZfS6L

**Comment by thesighguy** [comment]
Also, I was using v1 wormhole.

**Comment by leoluk** [comment, marked as answer]
The transaction was successfully processed by the Wormhole network, but the client-side transaction submission may have failed. Try clearing your cache and reloading https://v1.wormholebridge.com to see if the transaction can be completed there. Make sure both wallets are connected, and select both tokens from the picker.

If v1.wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

Note that Wormhole v2 was recently launched on wormholebridge.com, which fixes the bug you encountered! You will still have to use v1 to complete your in-progress transfer.

You can also try using the third-party https://www.interstellar.tk site to retry your VAA submission.

---

Digest (for reference only): 14d0d84c96aaf1b9299db39be1651d8742c07eec1236e6d9a70348f2e9128f1e

Signed VAA bytes to submit to the contract:

> 0x01000000010d004f4bb7b1e54222ef687dfd3a5249f51af9aa3fe324aebde3e6d3fa9ca2d5a0aa1db9bb779688e0cc9c88befa4571ab9cf59497d0fc4b714b38dd31ef65a0afa901015376ee9e7d827a9be69b4d4cfbef575b92537207b97c97dd548ebc6a1c50c0192aa82b0b75289cd2838015b496313603749e7188e81bfde20aa5a8728137c34e010345efb3dd8a72d102d7c07b28eedbf762ed524672d6999b09acdd7dd9c2ee34454c501f2af59e4495d83562081df6885814290f9b78a48213a95373d4296975f30105e16e1cb2a482eaa4a507c503974c30646b9b01c2c4438f32f04d37324974bf13434702a78f16c5ed6920049b165bb28fd1cb93c26bd49323fb7097cb357a0cee00061b5e80c9bf35ba6b251005eb31b5094bc309ace29820829eb52f0251df103db52432936a352a17f235ce070e049d6d73800632bbb64be584b6186450b16f26080007d25bef6b7b64a9c92af73fa892afede8b19e7a33f9b0422be21035940a0ca3465c4474f6a544d784e858405ff3b53bbddb54d2dae04e8ce8afdf3668fc02fec90108632b13d8beaff615fe52d92d18fae1159e7c8003fd8eabbc670f6a0b53ef076f2a837503ae57c252bcd51c2b1d9fd04d0073068c4f45d7634b7ddea111df492c010ad67b7d765a381b362d374b8eec465bb4ff1d7a14d67256e81dff7cc2dd20a41c1f71f22c5a7f1a42c182da2cf3191ea4d5a1136ca4004cca88524198a623a728000b7b326c539ac502cb5704b24ee7d2ab9aa054d26fbc7ad81a08c42c6d254a34e467450f813dd9315bb0459cd2608032b562a4b1ff35405dec0f6dafadd1e7086c010c6dbf9358a71ddac02eb182b49239c6bc9ab66537020e7a3a9a3bf5f0267012b35f8078bbfb8b40c345c4a5c5cefa241574a9f4dd4c6e5d663725dbf92afbda36000d9dd451f2b1cb2204a1ce2527ddf690cc7872070b873b749261d106a25f7f2c186188d099b9e55d30b9b568c70166af2dcbbe11336859390c2d3ae6dffe98c3570110dfe50a9314773c610f78cf85daa6934a43c71396da612d23583417a9a7bf17cd5b214280363fde50ca224e8b774ccdb0e083f4bf9d7fd08683e2d6d0674dfa6a00127ee5a3065d3d7e9174bb815b9d52f7cb85e9732a6dae0d4985dde46736ca849034715b71c52610e5d62934e32f6c7ad2bdbfd508b846b8c162c2884bacb9a08001614834641000000f950102d06458e7abe8fb2ccd42f2789f7643e226dc28752148f7c36a3736d8a1569ea9000000000000000000000000c1157c009f9ea8f6306b01a68ba421c77e9efd8a02000000000000000000000000d2877702675e6ceb975b4a1dff9fb7baf4c91ea9090000000000000000000000000000000000000000000000000000002c202e08f8
